Python/Python.sublime-syntax

```diff
@@ -15,38 +16,205 @@ file_extensions:
+      push:
+        - meta_scope: meta.statement.raise.python
+        - include: line_continuation_or_pop
+        - match: \b(from)\b
+          scope: keyword.control.flow.raise.from.python
+          set:
+            - meta_scope: meta.statement.raise.python
+            - include: line_continuation_or_pop
+            - include: expressions
+        - include: expressions
+    - match: \b(assert)\b
+      scope: keyword.other.assert.python
+      push:
+        - meta_scope: meta.statement.assert.python
+        - match: $
+          pop: true
+        - include: expressions
+    - match: \b(del)\b
+      scope: keyword.other.del.python
+      push:
+        - meta_scope: meta.statement.del.python
+        - include: line_continuation_or_pop
+        - include: expressions
+    - match: \b(print)\b(?! *($|[,.()\]}]))
+      scope: keyword.other.print.python
+      push:
+        - meta_scope: meta.statement.print.python
+        - include: line_continuation_or_pop
+        - include: expressions
+    - match: \b(exec)\b(?! *($|[,.()\]}]))
+      scope: keyword.other.exec.python
+      push:
+        - meta_scope: meta.statement.exec.python
+        - include: line_continuation_or_pop
+        - include: expressions
+    - match: \b(return)\b
+      scope: keyword.control.flow.return.python
+    - match: \b(break|continue|pass)\b
+      scope: keyword.control.flow.python
+
+  imports:
+    - match: \b(import)\b
+      scope: keyword.control.import.python
+      push:
+        - meta_scope: meta.statement.import.python
+        - include: line_continuation_or_pop
+        - include: import_alias
+        - include: dotted_name
+        - match: ','
+          scope: punctuation.separator.import-list.python
+    - match: \b(from)\b
+      scope: keyword.control.import.from.python
+      push:
+        - meta_scope: meta.statement.import.python
+          meta_content_scope: meta.import-source.python
+        - include: line_continuation_or_pop
+        - match: \b(import)\b
+          scope: keyword.control.import.python
+          set:
+            - include: line_continuation_or_pop
+            - match: ' *(\()'
+              captures:
+                1: punctuation.definition.begin.import-list
+              set:
+                - meta_scope: meta.statement.import.python
+                - include: comments
+                - match: \)
+                  scope: punctuation.definition.end.import-list
+                  pop: true
+                - include: import_name_list
+            - match: ''
+              set:
+                - meta_scope: meta.statement.import.python
+                - include: line_continuation_or_pop
+                - include: import_name_list
+        - include: dotted_name
+
+  import_name_list:
+    - match: ','
+      scope: punctuation.separator.import-list.python
+    - include: import_alias
+    - include: name
+    - match: \*
+      scope: keyword.other.import-all.python
```

Comment:
I was kinda at a loss here since there wasn't a scope for this previously and I didn't look into other languages either, although something with a "special meaning" should probably get a keyword scope and not constant. It really doesn't matter though, so I'll adjust to the current standard.